automation/check-patch.e2e-linux-bridge-functests.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -xeu
+
+# This script should be able to execute linux-bridge
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-linux-bridge-functests.sh
+
+teardown() {
+    cp $(find . -name "*junit*.xml") $ARTIFACTS || true
+    cluster/kubectl.sh logs -n cluster-network-addons -l app=cni-linux-bridge-plugin > $ARTIFACTS/linux-bridge.log || true
+    cluster/kubectl.sh get nodes -o yaml > $ARTIFACTS/nodes.yaml || true
+    rm -rf "${TMP_COMPONENT_PATH}"
+    cd ${TMP_PROJECT_PATH}
+    make cluster-down
+}
+
+build_binaries() {
+	./build_linux.sh
+}
+
+collect_binaries_shasum_from_repo() {
+	sha_file=$1
+	build_binaries
```

Comment:
This is a weird side-effect of a function that claims only to "collect". Could we decouple the two?

Reply:
Done